src/tools/compiletest/src/runtest.rs

```diff
@@ -359,6 +361,22 @@ impl<'test> TestCx<'test> {
         self.check_forbid_output(&output_to_check, &proc_res);
     }
 
+    fn run_crash_test(&self) {
+        let pm = self.pass_mode();
+        let proc_res = self.compile_test(WillExecute::No, self.should_emit_metadata(pm));
+        /*
+        eprintln!("{}", proc_res.status);
+        eprintln!("{}", proc_res.stdout);
+        eprintln!("{}", proc_res.stderr);
+        eprintln!("{}", proc_res.cmdline);
+        */
```

Comment:
commented out code

Reply:
I'd like to leave this in as this has been proven to be very useful when adding tests, to figure out why a test doesn't trigger an ice yet (due to RUSTFLAGs that compiletest sets by default).

In some tests you have to manually add some additional flags or enable optimizations/codegen for the ui test, where a regular `rustc $file` would already trigger the ice right away.

Reply:
could this then be some kind of compiletest flag or whatever so that the test writer can supply a flag to toggle verbose output like this, without having to comment/uncomment code and recompile compiletest?

Reply:
Probably, but seems out of scope for this pr.

Reply:
This is trivial to add. If you want to keep it in, put it behind a magic env var

Reply:
ah yeah env var sounds better, don't want to have to add new flags to compiletest/bootstrap rn :sweat_smile:

Reply:
I've gated it behind an env var `COMPILETEST_VERBOSE_CRASHES`